src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java

```diff
@@ -94,32 +110,26 @@ public OAuth2AccessToken readAccessToken(String token) {
     /**
      * Revokes the access tokens of the user with the given ID.
      *
-     * @param id
-     *            the user ID
-     * @param token
-     *            the token to access the service
+     * @param id    the user ID
+     * @param token the token to access the service
      */
     public void revokeAccessTokens(String id, String token) {
         AccessToken accessToken = new AccessToken.Builder(token).build();
-        OsiamConnector connector = createConnector();
         connector.revokeAllAccessTokens(id, accessToken);
     }
 
     private AccessToken validateAccessToken(String token) {
-        OsiamConnector connector = createConnector();
 
         AccessToken accessToken;
         try {
             accessToken = connector.validateAccessToken(new AccessToken.Builder(token).build());
+        } catch (ConnectionInitializationException e) {
+            String message = "Unable to retrieve access token from AuthServer " + authServerHome;
```

Comment:
Please write `auth-server` instead of `AuthServer` because this is the default name